File: Python/Arrays/problem_26. Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if(len(nums) == 0):
            return
        i = 0
        for j in range(len(nums)):
            if(nums[i] != nums[j]):
                i += 1
                nums[i] = nums[j]
        return i+1

File: Java/Arrays/problem_26. Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i = 0;
        if(nums.length == 0){
            return 0;
        }
        for(int j = 0; j < nums.length; j++){
            if(nums[i] != nums[j]){
                i++;
                nums[i] = nums[j];
            }
        }
        return i+1;
        
    }
}